### DISCUSSION TITLE
希望可以增加一个不生效的按键

### DISCUSSION THREAD

**Discussion by JzCherry** [shuax/MouseInc.Settings#486] [Ideas]
就比如ps里经常频繁使用alt键加鼠标右键来调整画笔的大小和硬度，用其他功能的时候又经常需要用到手势因此不能把ps加入到排除列表里，只能频繁的切换开关有点麻烦，因此需要一个按住alt或者ctrl键可以暂时停用功能的动作

**Comment by GOLDROBERT** [comment]
热心转发
https://www.zhihu.com/question/557457155/answer/2699425100
